elasticai/creator/nn/quantized_grads/fixed_point/_round_to_fixed_point_autograd.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+
+import torch
+
+from ._quantize_to_fixed_point import quantize
+from ._two_complement_fixed_point_config import FixedPointConfigV2
+
+
+class RoundToFixedPoint(torch.autograd.Function):
+    @staticmethod
+    def forward(ctx: Any, *args: Any, **kwargs: Any) -> torch.Tensor:
+        if len(args) != 3:
```

Comment:
you won't catch the case where people supply more parameters as kwargs, but I don't even think you need to handle that here

Reply:
I added the catch